deploy/addons/csi-hostpath-driver/deploy/csi-hostpath-plugin.yaml.tmpl

```diff
@@ -16,64 +16,37 @@
 # are needed only because of condition explained in
 # https://github.com/kubernetes/kubernetes/issues/69608
 
-kind: Service
-apiVersion: v1
-metadata:
-  name: csi-hostpathplugin
-  namespace: kube-system
-  labels:
-    app: csi-hostpathplugin
-    addonmanager.kubernetes.io/mode: Reconcile
-spec:
-  selector:
-    app: csi-hostpathplugin
-  ports:
-    - name: dummy
-      port: 12345
----
-kind: StatefulSet
+kind: DaemonSet
 apiVersion: apps/v1
 metadata:
   name: csi-hostpathplugin
   namespace: kube-system
   labels:
+    app.kubernetes.io/instance: hostpath.csi.k8s.io
+    app.kubernetes.io/part-of: csi-driver-host-path
+    app.kubernetes.io/name: csi-hostpathplugin
+    app.kubernetes.io/component: plugin
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
-  serviceName: "csi-hostpathplugin"
-  # One replica only:
-  # Host path driver only works when everything runs
-  # on a single node. We achieve that by starting it once and then
-  # co-locate all other pods via inter-pod affinity
-  replicas: 1
   selector:
     matchLabels:
-      app: csi-hostpathplugin
+      app.kubernetes.io/instance: hostpath.csi.k8s.io
+      app.kubernetes.io/part-of: csi-driver-host-path
+      app.kubernetes.io/name: csi-hostpathplugin
+      app.kubernetes.io/component: plugin
       addonmanager.kubernetes.io/mode: Reconcile
   template:
     metadata:
       labels:
-        app: csi-hostpathplugin
+        app.kubernetes.io/instance: hostpath.csi.k8s.io
+        app.kubernetes.io/part-of: csi-driver-host-path
+        app.kubernetes.io/name: csi-hostpathplugin
+        app.kubernetes.io/component: plugin
         addonmanager.kubernetes.io/mode: Reconcile
         kubernetes.io/minikube-addons: csi-hostpath-driver
     spec:
-      serviceAccount: csi-external-health-monitor-controller
+      serviceAccount: csi-hostpathplugin-sa
       containers:
-        - name: csi-external-health-monitor-agent
```

Comment:
health-agent was merged to kubelet: https://github.com/kubernetes-csi/external-health-monitor#overview